src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -982,6 +985,53 @@ def load_iv_iron_hemoglobin_effect_size(
     return data
 
 
+def _add_hemoglobin_exposure_start_and_end(df: pd.DataFrame) -> pd.DataFrame:
+    df["hemoglobin_exposure_start"] = df["exposure"]
+    df["hemoglobin_exposure_end"] = df["exposure"][1:].tolist() + [np.inf]
+    return df
+
+
+def load_iv_iron_lbwsg_effect_size(
+    key: str, location: str, years: Optional[Union[int, str, List[int]]] = None
+) -> pd.DataFrame:
+    index_cols = ["sex", "outcome", "hemoglobin_exposure_start", "hemoglobin_exposure_end"]
+    data_dir = paths.HEMOGLOBIN_EFFECTS_DATA_DIR / "iv_iron_lbwsg_shifts"
+    filepaths = glob.glob(str(data_dir / "*.csv"))
+
+    data = pd.concat([pd.read_csv(filepath) for filepath in filepaths])
+    data = data.loc[data["location"] == location.lower()].drop("location", axis=1)
+
+    data = data.sort_values(["draw", "outcome", "sex", "exposure"])
+    data = data.groupby(["draw", "outcome", "sex"]).apply(
```

Comment:
I don't see why we need to apply this to the groups. It probably doesn't matter much, but it makes the code more confusing to read. You didn't need to do it for the stillbirth data.